### PR TITLE
navigationUpdate

### PR DESCRIPTION
**Pull request by MaxMer25**
I updated the navigation using the useRouter Hook

### CODE REVIEW

File: components/Navigation/Navigation.js [hunk 1]
```diff
@@ -22,18 +18,12 @@ export default function Navigation({currentPage, setCurrentPage}) {
...
-          <StyledLink
-            onClick={() => setCurrentPage("home")}
-            href={"/home"}
-            className={
-              currentPage === "home" ? "highlightedIcon" : "normalIcon"
-            }
-          >
+          <StyledLink onClick={() => router.push("/home")} href={"/home"}>
             <svg
               width="62"
```
Comment (actyralf) [inline components/Navigation/Navigation.js:31]: Please add a `<title>` tag to your SVGs to make them accessible (https://css-tricks.com/accessible-svgs/).
You can test accessibility with the Lighthouse tool in the Chrome Dev Tools.

**Review by actyralf — CHANGES_REQUESTED** [review]
Very good!
One more thing though... (see comment below)

File: components/Navigation/Navigation.js (hunk 1 above)
Comment (MaxMer25) [reply components/Navigation/Navigation.js:31]: Hey Ralf, thanks for the Review. I updated it.

**Review by actyralf — APPROVED** [review]
Great work!

**Pull request merged**